pyproject.toml

```diff
@@ -25,11 +25,14 @@ classifiers = [
     "Topic :: Scientific/Engineering :: Visualization",
 ]
 dependencies = [
-    "cvxopt>=1.2.0",
+    "attrs>=23.1.0",
+    "click>=8.1.6",
+    "cvxopt>=1.3.1",
     "h5py>=2.10.0",
     "numpy>=1.17.3",
+    "pandas>=2.0.3",
```

Comment:
Feels like we may as well require it? https://github.com/quantumjot/btrack/blob/e7501297b8735c4aabae19428371c71fd8298d60/btrack/btypes.py#L576

Reply:
I think it's time